global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
     "version": "6.0.402",
```

Comment:
I'm super-ignorant about the finer details of 'global.json'

Can you explain what the fixes do?

Reply:
I only have .NET 7 installed locally to save disk space. The change allows me to use that instead of the v6 framework. Should change that back before merging.

Reply:
Following up on this -- does it need to be changed back before merging?